pkg/controller/shared/shared_informer.go

```diff
@@ -175,6 +176,10 @@ func (f *sharedInformerFactory) KubernetesInformers() informers.SharedInformerFa
 	return kubernetesSharedInformer{f}
 }
 
+func (f *sharedInformerFactory) ServiceAccounts() ServiceAccountInformer {
```

Comment:
Add here todo, with a link to you upstream PR, so that we know we should use the upstream version upon next rebase (to 1.5).